src/pages/ProductDetail/ProductDetail.js

```diff
@@ -1,14 +1,27 @@
 import React, { useEffect, useState } from 'react';
-import { useNavigate } from 'react-router-dom';
+import { useNavigate, useParams } from 'react-router-dom';
+import { AiOutlineHeart, AiTwotoneHeart } from 'react-icons/ai';
 import Recommend from './component/Recommend';
 import './ProductDetail.scss';
 
 const ProductDetail = () => {
   const [recommendList, setRecommendList] = useState([]);
   const [productList, setProductList] = useState({});
   const [count, setCount] = useState(1);
+  const [products, setProducts] = useState([{}]);
```

Comment:
```suggestion
  const [products, setProducts] = useState({});
```
보통 디테일 페이지에는 단일 데이터가 들어오게 되는 경우기 때문에
초기값을 빈 객체로 설정하는 것이 좋습니다

Reply:
![스크린샷 2023-06-05 오후 12 11 33](https://github.com/wecode-bootcamp-korea/46-1st-BestFriend-frontend/assets/123567359/1053f26e-ad9a-4e89-a0cd-bdf2a1da34ab)
fetch(GET) 요청으로 받아온값을 setProducts(data.product) 형식으로 넣은것인데 BE 에서 보내주는 형식이 위 사진과 같은 경우에도 
[{}] 형태가 아닌 {} 형태로 해도 되는것인지 궁금합니다!

Reply:
결국 단일 값이 담긴 상태에서 적용해야하기 때문에
통신후 받아 온 값을 setProducts(data.product[0]) 으로 전달하면
하나의 객체 값을 전달해 줄 수 있고 초기값을 {} 로 할 수 있겠네요!